wicket/src/widgets/rack.rs

```diff
@@ -255,27 +264,35 @@ impl RackState {
     /// Return true if the component being hovered over changed, false otherwise. This
     /// allows us to limit the number of re-draws necessary.
     pub fn set_hover_state(&mut self, x: u16, y: u16) -> bool {
-        // Find the interesecting component.
-        // I'm sure there's a faster way to do this with percentages, etc..,
-        // but this works for now.
-        for (id, rect) in self.component_rects.iter() {
-            let mouse_pointer = Rect { x, y, width: 1, height: 1 };
-            if rect.intersects(mouse_pointer) {
-                if self.hovered == Some(*id) {
-                    // No chnage
-                    return false;
+        match &self.component_rects {
+            ComponentRects::Displayed { rects_map, .. } => {
+                // Find the interesecting component.
+                // I'm sure there's a faster way to do this with percentages, etc..,
+                // but this works for now.
+                for (id, rect) in rects_map {
+                    let mouse_pointer = Rect { x, y, width: 1, height: 1 };
+                    if rect.intersects(mouse_pointer) {
+                        if self.hovered == Some(*id) {
+                            // No chnage
```

Comment:
I know it's my typo, but I just noticed. Feel free to fix or leave for later :)

Reply:
Fixed.